confectory-core/src/main/java/net/obvj/confectory/Confectory.java

```diff
@@ -0,0 +1,49 @@
+package net.obvj.confectory;
+
+/**
+ * A default {@link ConfigurationContainer}
```

Comment:
This class is intended for more than that. I would suggest something like:
"A Facade for common operations in the {@code Confectory} project."